lib/modules/manager/sbt/extract.spec.ts

```diff
@@ -433,9 +472,308 @@ describe('modules/manager/sbt/extract', () => {
+              datasource: 'sbt-package',
+              depName: 'org.scalatest:scalatest-funsuite',
+              depType: 'Test',
+              editFile: `project/Versions.scala`,
+              fileReplacePosition: 13,
+              groupName: 'scalaTest',
+              packageName: 'org.scalatest:scalatest-funsuite_2.13',
+              registryUrls,
+              variableName: 'scalaTest',
+            },
+            {
+              currentValue: '1.17.5',
+              datasource: 'sbt-package',
+              depName: 'org.mockito:mockito-scala-scalatest',
+              depType: 'Test',
+              editFile: `project/Versions.scala`,
+              fileReplacePosition: 14,
+              groupName: 'mockito',
+              packageName: 'org.mockito:mockito-scala-scalatest_2.13',
+              registryUrls,
+              variableName: 'mockito',
+            },
+            {
+              currentValue: '3.1.9',
+              datasource: 'sbt-package',
+              depName:
+                'com.softwaremill.sttp.client3:async-http-client-backend-future',
+              depType: 'Test',
+              editFile: `project/Versions.scala`,
+              fileReplacePosition: 4,
+              groupName: 'sttp',
+              packageName:
+                'com.softwaremill.sttp.client3:async-http-client-backend-future_2.13',
+              registryUrls,
+              variableName: 'sttp',
+            },
+          ],
+          packageFile: `project/Versions.scala`,
+        },
+        {
+          deps: [
+            {
+              currentValue: '1.9.3',
+              datasource: 'sbt-plugin',
+              depName: 'org.scoverage:sbt-scoverage',
+              depType: 'plugin',
+              packageName: 'org.scoverage:sbt-scoverage',
+              registryUrls: [
+                ...registryUrls,
+                'https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases',
+              ],
+            },
+            {
+              currentValue: '1.0.1',
+              datasource: 'sbt-plugin',
+              depName: 'com.typesafe:sbt-mima-plugin',
+              depType: 'plugin',
+              packageName: 'com.typesafe:sbt-mima-plugin_2.13',
+              registryUrls: [
+                ...registryUrls,
+                'https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases',
+              ],
+            },
+            {
+              currentValue: '0.4.3',
+              datasource: 'sbt-plugin',
+              depName: 'pl.project13.scala:sbt-jmh',
+              depType: 'plugin',
+              packageName: 'pl.project13.scala:sbt-jmh',
+              registryUrls: [
+                ...registryUrls,
+                'https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases',
+              ],
+            },
+          ],
+          packageFile: 'project/plugins.sbt',
+        },
+        {
+          deps: [
+            {
+              currentValue: undefined,
```

Comment:
found some bug here let me fix it

Reply:
fixed
[fixed by adding `var` keyword support ](https://github.com/renovatebot/renovate/pull/21622/commits/240ac8d5e909de726f432b50b5ac3d9086477740)